pkg/types/validation/installconfig.go

```diff
@@ -476,34 +476,49 @@ func validateVIPsForPlatform(network *types.Networking, platform *types.Platform
 	}
 	switch {
 	case platform.BareMetal != nil:
+		ensureIPv4IsFirstInDualStackSlice(&platform.BareMetal.APIVIPs)
```

Comment:
The error reported by ensureIPv4IsFirstInDualStackSlice() is not bubbled up via allErrs. Is that by design?